draft-ietf-ppm-dap.md

```diff
@@ -1189,7 +1190,14 @@ Let `[aggregator]` denote the helper's API endpoint. The leader sends a POST
 request to `[aggregator]/aggregate` with its AggregateInitializeReq message as
 the payload. The media type is "application/dap-aggregate-initialize-req".
 
-#### Helper Initialization
+The Helper's response will be an `AggregationJob` message (see
```

Comment:
```suggestion
The Helper's response will be an `AggregateInitializeResp` message (see
```

Reply:
`AggregationJob` is correct, I forgot to rebase this on `main` after merging #398.